contents/docs/self-host/configure/migrate-postgres-clickhouse.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: Migrate from Postgres to ClickHouse
+sidebar: Docs
+showTitle: true
+---
+
+> This documentation is in **Beta**. Please provide feedback via the [PostHog Communty Slack workspace](/slack) or [via a GitHub issue](https://github.com/PostHog/posthog.com/issues).
+
+**PostHog backed by Postgres is now deprecated**. We will continue to provide support to Postgres backed installs but we strongly encourage you to migrate to PostHog backed by ClickHouse for improved performance, to receive new features, and continued support in the future.
+
+## Before you begin
+
+TODOs:
+- Is there a minimum version for PostHog + Postgres and a minimum version for PostHog + ClickHouse? For example, does the Replicator plugin have a minimum PostHog version to work within?
+
+In order to migrate you must have the following in place:
+
+1. The PostHog + Postgres installation
+2. A new PostHog + ClickHouse installation
+
+## Export
+
+TODOs:
+- Are these the correct tables to dump?
+- Should we be using `--data-only` to only dump the data and not the schema creation?
+
+The first step in the migration process is to create a Postgres database dump. With a PostHog + Postgres instance up and running, run the `pg_dump` command with the following format:
+
+```
+$ pg_dump -d [database] -h [host] -U [user] -T posthog_event -T posthog_person -T posthog_sessionrecordingevent -f [output-file]
+```
+
+For example:
+
+```
+$ pg_dump -d posthog -h localhost -U posthog -T posthog_event -T posthog_person -T posthog_sessionrecordingevent -f posthog-migration.sql
+```
+
+The example command above creates a `posthog-migration.sql` that contains SQL statements to create the schema and insert the data data, excluding event data, from the existing Postgres database.
+
+## Import
+
```

Comment:
suggestion: add note about how the pg_dump method will only work with a full db override.

Reply:
So, the `pg_restore` command should use the `--clean` flag? 

Found via https://stackoverflow.com/a/43603983/39904

Reply:
Maybe we should tell people to use `--data-only` here as well as just add a note that the DB should be clean of data in advance